src/graphql/public/root/subscription/ln-invoice-payment-status.ts

```diff
@@ -50,6 +50,8 @@ const LnInvoicePaymentStatusSubscription = {
   },
 
   subscribe: async (_source: unknown, args: LnInvoicePaymentSubscribeArgs) => {
+    baseLogger.info("Running subscribe: ln-invoice-payment-status")
```

Comment:
Leaving for now. Can be used for integration testing, but can eventually be removed